es-ds-docs/pages/molecules/skeleton.vue

```diff
@@ -0,0 +1,164 @@
+];
+
+const { $prism } = useNuxtApp();
+const compCode = ref('');
+const docCode = ref('');
+onMounted(async () => {
+    if ($prism) {
+        const compSource = await import('@energysage/es-ds-components/components/es-skeleton.vue?raw');
+        // eslint-disable-next-line import/no-self-import
+        const docSource = await import('./skeleton.vue?raw');
+        compCode.value = $prism.normalizeCode(compSource.default);
+        docCode.value = $prism.normalizeCode(docSource.default);
+        $prism.highlight();
+    }
+
+    startLoading();
+});
+</script>
+
+<template>
+    <div>
+        <h1>Skeleton</h1>
+        <p>
+            Extended from
+            <nuxt-link
+                href="https://v3.primevue.org/skeleton/"
+                target="_blank">
+                PrimeVue Skeleton
+            </nuxt-link>
+            and
+            <nuxt-link
+                href="https://bootstrap-vue.org/docs/components/skeleton/"
+                target="_blank">
+                Bootstrap Skeleton
+            </nuxt-link>
+            styles
+        </p>
+
+        <h2>Basic examples</h2>
+        <p>Wave Animation:</p>
+        <es-row class="mb-200">
+            <es-col lg="6">
+                <es-skeleton />
+                <es-skeleton width="75%" />
+                <es-skeleton height="3rem" />
+            </es-col>
+            <es-col
+                lg="6"
+                class="d-flex">
+                <es-skeleton size="5rem" />
+                <es-skeleton
+                    size="4rem"
+                    class="rounded-circle ml-100" />
+            </es-col>
+        </es-row>
+
+        <p>Fade Animation:</p>
+        <es-row class="mb-500">
+            <es-col lg="6">
+                <es-skeleton
+                    animation="fade"
+                    height="33px"
+                    width="100%" />
+                <es-skeleton
+                    animation="fade"
+                    height="25px"
+                    width="100%" />
+            </es-col>
+        </es-row>
+
+        <h2>Helper components</h2>
+
+        <h3>Skeleton Wrapper</h3>
+        <es-row class="mb-500 d-flex">
+            <es-col lg="2">
+                <es-button @click="startLoading()">Reload content</es-button>
+            </es-col>
+            <es-col
+                lg="8"
+                class="ml-100">
+                <es-skeleton-wrapper :loading="loading">
+                    <template #loading>
+                        <es-skeleton width="85%"></es-skeleton>
+                        <es-skeleton width="55%"></es-skeleton>
+                    </template>
+
+                    <p>
+                        Lorem ipsum dolor sit amet, consectetur adipiscing elit. Maecenas viverra nunc sapien, non
+                        rhoncus elit tincidunt vitae.
+                    </p>
+                </es-skeleton-wrapper>
+            </es-col>
+        </es-row>
+
+        <h3>Skeleton Image</h3>
```

Comment:
Nit: sentence case to match our content guidelines on headings/labels - "Skeleton image"